### ISSUE TITLE
Attempted import error: 'blockies-react-svg' does not contain a default export (imported as 'BlockiesSvg').

### ISSUE THREAD

**Issue by mwawrusch** [YYsuni/blockies-react-svg#1]
Hi,
thanks for the great library. I ran into a small issue in a next.js v13 app, when using the lib I get the following error:

Attempted import error: 'blockies-react-svg' does not contain a default export (imported as 'BlockiesSvg').

Using it like this:
```
import BlockiesSvg from 'blockies-react-svg'
```

```
<BlockiesSvg 
  address={address.toLowerCase()}
  className="identicon" 
  />
```

tsconfig:

```
{
  "compilerOptions": {
    "target": "es5",
    "lib": ["dom", "dom.iterable", "esnext"],
    "allowJs": true,
    "skipLibCheck": true,
    "strict": true,
    "forceConsistentCasingInFileNames": true,
    "noEmit": true,
    "esModuleInterop": true,
    "module": "esnext",
    "moduleResolution": "node",
    "resolveJsonModule": true,
    "isolatedModules": true,
    "jsx": "preserve",
    "incremental": true,
    "plugins": [
      {
        "name": "next"
      }
    ],
    "baseUrl": ".",
    "paths": {
      "@/*": ["./*"]
    }
  },
  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts", "app/[walletId]/currencies"],
  "exclude": ["node_modules"]
}
```

Any idea what might cause this?

**Comment by YYsuni** [comment]
Thank you for bringing this to my attention. I haven't used it in Next.js before, but I will do my best to fix the issue.

**Comment by YYsuni** [comment]
Hi, I have fixed the issue in v0.0.12 by clarifying Next.js' module resolution. I'm not sure if this will work for your issue because the problem in my new Next.js 13 app is not the same as the error you reported.

You can try it now, and feel free to ask me any questions.

**Comment by mwawrusch** [comment]
Error message is gone, thank you!
